tests/data/permutations/ru_permutations_mathtools.csv

```diff
@@ -0,0 +1,24 @@
+number,text
```

Comment:
@arnavkapoor I spot-checked a few cases from this list and look looked good, please tell if you have any doubts here.